examples/kafka-hub/load_balancer/lb_service.bal

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+service /hub on new http:Listener(LB_PORT,
+    secureSocket = {
+        key: {
+            certFile: "./resources/server.crt",
+            keyFile: "./resources/server.key"
+        }
+    }
+) {
```

Comment:
Wouldn't it look better to define the listener as a separate variable for better readability?

Reply:
Fixed with [cbb535d29b1f96ef61b1e1bb8fc5b0dfa2ab7078](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/876/commits/cbb535d29b1f96ef61b1e1bb8fc5b0dfa2ab7078)